Makefile

```diff
@@ -0,0 +1,21 @@
+vendor:
```

Comment:
Do you plan to include a `help` entry, similar to complyctl and a couple of other repositories?